Sources/Transitions/TransitionExtentions.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright © 2019 SkeletonView. All rights reserved.
+
+import UIKit
+
+
+internal extension UIView {
```

Comment:
by default all classes, extension, struct, etc are `internal`, so we don't need to specify it :)